src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -532,17 +529,17 @@ def post_process_panoptic_segmentation(
                 for k in range(pred_labels.shape[0]):
                     pred_class = pred_labels[k].item()
                     # check if pred_class is not a "thing", so it can be merged with other instance. For example, class "sky" cannot have more then one instance
-                    is_stuff = not is_thing_map[pred_class]
+                    should_fuse = pred_class in label_ids_to_fuse
```

Comment:
This will crash because you are not setting `label_ids_to_fuse` to an empty set anymore.
And you cannot use it as a default argument because `set` is mutable.

Reply:
Fixed, setting `should_fuse` to `False` and checking if `label_ids_to_fuse is not None`